src/scenes/Home/TerrawareHomeView/PlantingSiteStats.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Icon } from '@terraware/web-components';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+import Link from 'src/components/common/Link';
+import PlantingSiteSelector from 'src/components/common/PlantingSiteSelector';
+import { APP_PATHS } from 'src/constants';
+import { selectLatestObservation } from 'src/redux/features/observations/observationsSelectors';
+import { selectPlantingSites } from 'src/redux/features/tracking/trackingSelectors';
+import { useAppSelector } from 'src/redux/store';
+import strings from 'src/strings';
+import { PlantingSite } from 'src/types/Tracking';
+import useMapboxToken from 'src/utils/useMapboxToken';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+
+import StatsCardItem from './StatsCardItem';
+
+export const PlantingSiteStats = () => {
+  const { isDesktop } = useDeviceInfo();
+  const theme = useTheme();
+  const navigate = useNavigate();
+  const plantingSites = useAppSelector(selectPlantingSites);
+  const { token } = useMapboxToken();
+  const defaultTimeZone = useDefaultTimeZone();
+
+  const [selectedPlantingSiteId, setSelectedPlantingSiteId] = useState<number>();
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+
+  const observation = useAppSelector((state) =>
+    selectLatestObservation(state, selectedPlantingSiteId || -1, defaultTimeZone.get().id)
+  );
+
+  const geojson = {
+    type: 'FeatureCollection',
+    features: [
+      {
+        type: 'Feature',
+        geometry: selectedPlantingSite?.boundary,
+        properties: {},
```

Comment:
I was able to get fillColor + any other style specs by adding them to the properties, specs here: https://github.com/mapbox/simplestyle-spec/blob/master/1.1.0/README.md

From some playing around, it looks like this needs to happen at a feature level but I am not entirely sure. 

For example: With a red fill color: I added `fill: "#FF0000"`

https://api.mapbox.com/styles/v1/mapbox/satellite-streets-v12/static/geojson(%7B%22type%22%3A%22FeatureCollection%22%2C%22features%22%3A%5B%7B%22type%22%3A%22Feature%22%2C%22geometry%22%3A%7B%22type%22%3A%22MultiPolygon%22%2C%22coordinates%22%3A%5B%5B%5B%5B-120.99817678%2C44.00783718%5D%2C%5B-120.99817678%2C43.98678116%5D%2C%5B-121.00906272%2C43.9787617%5D%2C%5B-121.02189882%2C43.98518127%5D%2C%5B-121.04330608%2C43.9843094%5D%2C%5B-121.0614078%2C43.98212755%5D%2C%5B-121.07752025%2C43.98991393%5D%2C%5B-121.08695438%2C44.00783718%5D%2C%5B-121.0639738%2C44.01462225%5D%2C%5B-121.04848173%2C44.01802644%5D%2C%5B-121.05653535%2C44.0327989%5D%2C%5B-121.03657854%2C44.04540623%5D%2C%5B-121.00609904%2C44.02532019%5D%2C%5B-120.99817678%2C44.00783718%5D%5D%5D%5D%7D%2C%22properties%22%3A%7B%22fill%22%3A%22%23FF0000%22%7D%7D%5D%7D)/auto/580x360@2x?padding=100&access_token=tk.eyJ1IjoidGVycmFmb3JtYXRpb24iLCJleHAiOjE3Mjk4MDY1MjAsImlhdCI6MTcyOTgwNDcyMCwic2NvcGVzIjpbInN0eWxlczp0aWxlcyIsInN0eWxlczpyZWFkIiwiZm9udHM6cmVhZCJdLCJjbGllbnQiOiJhcGkifQ.TV3rjLXh_WSzxowRhPSRKw

Reply:
Hope this simplifies some API you have to do!

Reply:
I'm not sure why it didn't appear to be working when I first tried setting styles via `properties` at the feature level, but I was able to get it working correctly with your example. Thanks Tommy! 🙌 💯